appveyor.yml

```diff
@@ -125,13 +125,13 @@ for:
     - sh: sudo apt-get install -y git wget qemu qemu-system qemu-system-x86
 
     before_build:
-    - sh: dotnet restore $APPVEYOR_BUILD_FOLDER/Source/Mosa.LinuxCI.sln
+    - sh: dotnet restore $APPVEYOR_BUILD_FOLDER/Source/Mosa.sln
```

Comment:
This should be changed back to use `Mosa.Linux.sln` as the normal solution uses windows-only components that can't be compiled on linux.